mailjet_rest/client.py

```diff
@@ -9,12 +9,19 @@
 
 requests.packages.urllib3.disable_warnings()
 
+
 class Config(object):
-    API_URL = 'https://api.mailjet.com/v3/'
-    API_DOC = 'http://dev.mailjet.com/email-api/v3/'
+    API_URL = 'https://api.mailjet.com/'
+    API_DOC = 'http://dev.mailjet.com/email-api/'
+    version = 'v3/'  # urljoin requires forward slash
```

Comment:
@jorgii as per our discussions, the trailing `/` shouldn't be part of the version but added when building the URL